net/suricata/Makefile

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright (C) 2018 Ashkan Jazayeri <ashkan@jazayeri.net>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=suricata
+PKG_VERSION:=4.0.4
+PKG_RELEASE:=1
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=\
+		https://github.com/OISF/suricata/releases/download/$(PKG_NAME)-$(PKG_VERSION)/ \
+		https://www.openinfosecfoundation.org/download/
+
+PKG_HASH:=617e83b6e20b03aa7d5e05a980d3cb6d2810ec18a6f15a36bf66c81c9c0a2abb
+PKG_FIXUP:=autoreconf
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+PKG_LICENSE:=GPL-2.0
+PKG_MAINTAINER:=Ashkan Jazayeri <ashkan@jazayeri.net>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/suricata
+ SUBMENU:=Firewall
+ SECTION:=net
+ CATEGORY:=Network
+ DEPENDS:=+jansson +libiconv-full +libpcap +libpcre +libpthread +libyaml 
+ DEPENDS+=+zlib +SURICATA_LOGROTATE:logrotate +SURICATA_LUA:lua +SURICATA_MAGIC:file 
+ DEPENDS+=+SURICATA_NFLOG:libnetfilter-log +SURICATA_NFQ:kmod-ipt-nfqueue +SURICATA_NFQ:libnetfilter-queue
+ TITLE:=Network IDS, IPS and NSM engine
+ URL:=https://github.com/OISF/suricata
+endef
+
+define Package/suricata/description
+ Suricata is a network IDS, IPS and NSM engine.
+endef
+
+define Package/suricata/conffiles
+/etc/config/suricata
+/etc/logrotate.d/suricata.logrotate
+/etc/suricata/
+endef
+
+define Package/suricata/config
+	source "$(SOURCE)/Config.in"
+endef
+
+TARGET_CPPFLAGS += -I$(STAGING_DIR)/usr/lib/libiconv-full/include
+TARGET_LDFLAGS += -L$(STAGING_DIR)/usr/lib/libiconv-full/lib
+
+CONFIGURE_ARGS += \
+	$(if $(CONFIG_SURICATA_DEBUG),--enable,--disable)-debug \
+	--disable-gccmarch-native \
+	$(if $(CONFIG_SURICATA_MAGIC),--enable-magic,--disable-libmagic) \
+	--disable-python \
+	--disable-static \
+	$(if $(CONFIG_SURICATA_SOCKET),--enable,--disable)-unix-socket
+
+ifeq ($(CONFIG_SURICATA_LUA),y)
+CONFIGURE_ARGS += \
+	--enable-lua
+endif
+ifeq ($(CONFIG_SURICATA_NFLOG),y)
+CONFIGURE_ARGS += \
+	--enable-nflog
+endif
+ifeq ($(CONFIG_SURICATA_NFQ),y)
+CONFIGURE_ARGS += \
+	--enable-nfqueue
+endif
+ifeq ($(CONFIG_SURICATA_PIE),y)
```

Comment:
This should not even be an option. There's PKG_ASLR_PIE that sets this globally.